Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -102,10 +104,11 @@ class FontDescription {
             variantEastAsianVariant(),
             variantEastAsianWidth(),
             variantEastAsianRuby(),
-            variantEmoji() };
+            variantEmoji()
+        };
     }
     FontOpticalSizing opticalSizing() const { return static_cast<FontOpticalSizing>(m_opticalSizing); }
-    FontStyleAxis fontStyleAxis() const { return m_fontStyleAxis ? FontStyleAxis::ital : FontStyleAxis::slnt; }
+    FontStyleAxis fontStyleAxis() const { return static_cast<FontStyleAxis>(m_fontStyleAxis); }
```

Comment:
Kind of wish there was a special purpose casting function for this instead of just a straight `static_cast`; kind of a counterpart to enumToUnderlyingType just for clarity and could be restricted in types more than `static_cast` is.

Reply:
Yeah, an `enumCast<>` or something?